test_requirements.txt

```diff
@@ -1,6 +1,7 @@
 coveralls
+flake8
+pysatSpaceWeather
 pytest-cov
-pytest-flake8
 pytest-ordering
 sphinx
 sphinx_rtd_theme
```

Comment:
The stuff below this is also not alphabetical

Reply:
🙈